File: MetodoBinding/app/src/main/java/com/devheverton/metodobinding/MainActivity.java
package com.devheverton.metodobinding;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

import com.devheverton.metodobinding.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.btnCadastrar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                binding.txtMensagem.setText("O Usuário foi Cadastrado com sucesso");
            }
        });
    }
}

File: MeuPortiflio/app/src/main/java/com/devheverton/meuportiflio/MainActivity.kt
package com.devheverton.meuportiflio

import android.content.Intent
import android.graphics.BitmapFactory
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.core.graphics.drawable.RoundedBitmapDrawable
import androidx.core.graphics.drawable.RoundedBitmapDrawableFactory
import com.devheverton.meuportiflio.databinding.ActivityMainBinding

private lateinit var binding: ActivityMainBinding

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        val fotoPerfil = BitmapFactory.decodeResource(resources,R.drawable.foto_perfil)
        val circulo = RoundedBitmapDrawableFactory.create(resources,fotoPerfil)
        circulo.isCircular = true
        binding.imgFotoPerfil.setImageDrawable(circulo)

        binding.btProjetos.setOnClickListener {
           val intent = Intent(this,Projetos::class.java)
            startActivity(intent)

            }

        binding.btContato.setOnClickListener {
            val intent = Intent(this,Contato::class.java)
            startActivity(intent)
        }

        }

}